frontend/vite.config.ts

```diff
@@ -4,6 +4,9 @@ import checker from 'vite-plugin-checker'
 
 // https://vitejs.dev/config/
 export default defineConfig({
+  server: {
```

Comment:
In addition, can we add the following lines to ensure that the build preview will also be run on the same port?

```
preview: {
  3000
}
```